python/lsst/pipe/tasks/postprocess.py

```diff
@@ -244,15 +247,16 @@ def run(self, catalog, visit, detector, **kwargs):
         -------
         result : `~lsst.pipe.base.Struct`
             ``table``
-                `DataFrame` version of the input catalog
+                `astropy.table.Table` version of the input catalog
         """
         self.log.info("Generating DataFrame from src catalog visit,detector=%i,%i", visit, detector)
-        df = catalog.asAstropy().to_pandas().set_index("id", drop=True)
-        df["visit"] = visit
+        tbl = catalog.asAstropy()
+        tbl.add_index("id")
```

Comment:
This makes me nervous.  Why do we need the index here?  We aren't persisting it... Also astropy indexes are wonky and I don't want to rely on them since the support seems nonexistent.